xtable-core/src/main/java/org/apache/xtable/delta/DeltaStatsExtractor.java

```diff
@@ -74,9 +80,13 @@ public class DeltaStatsExtractor {
 
   private static final DeltaStatsExtractor INSTANCE = new DeltaStatsExtractor();
 
-  private static final String PATH_DELIMITER = "\\.";
   private static final ObjectMapper MAPPER = new ObjectMapper();
 
+  /* this data structure collects type names of all unrecognized Delta Lake stats. For instance
+  data file stats in presence of delete vectors would contain 'tightBounds' stat which is
+  currently not handled by XTable */
+  private static final Set<String> unsupportedStats = new HashSet<>();
```

Comment:
Should this be an instance variable instead? if not, let's follow the naming conventions and use `UPPER_UNDERSCORE`